hangups/ui/__main__.py

```diff
@@ -903,6 +903,9 @@ def main():
                   help='keybinding for alternate up key')
     key_group.add('--key-down', default='j',
                   help='keybinding for alternate down key')
+    notification_group = parser.add_argument_group('Notifications')
+    notification_group.add('-n', '--notification-type', default='full',
+                      help='choose notification type (full, none or discreet)')
```

Comment:
Do you think it would be better to have separate `--disable-notifications` and `--discreet-notifications` options? If we want to support another notification type later, we'd need discreet and non-discreet variants for each (eg.`type-a`, `type-a-discreet`, `type-b`, `type-b-discreet`).